### ISSUE TITLE
Image width and height always stays equal to imagesInitialDimensions prop

### ISSUE THREAD

**Issue by KristineTrona** [archriss/react-native-render-html#311]
### Is this a bug report or a feature request?

This is a bug report.

In ``/src/HTMLImage.js`` the imagesInitialDimensions are never overwriten by the setState call in getImageSize() and always stay to the default prop values. 

The reason for this is that ``this.mounted`` is expected to be ``true``, otherwise this.setState is not called. In componentDidMount() the ``this.mounted`` value is set to ``true`` only after this.getImageSize() is called, which is why the ``setState()`` is never executed.

```
    if (styleWidth && styleHeight) {
      return (
        this.mounted &&
        this.setState({
          width:
            typeof styleWidth === 'string' && styleWidth.search('%') !== -1
              ? styleWidth
              : parseInt(styleWidth, 10),
          height:
            typeof styleHeight === 'string' && styleHeight.search('%') !== -1
              ? styleHeight
              : parseInt(styleHeight, 10)
        })
      );
    }
```

### Fix
Simply set ``this.mounted`` to ``true`` before calling this.getImageSize()

```
  componentDidMount() {
    this.mounted = true;
    this.getImageSize();
  }

```

**Comment by JoeDuncko** [comment]
I'm having this issue as well, any chance you can make a PR with that fix?

**Comment by JoeDuncko** [comment]
Actually, https://github.com/archriss/react-native-render-html/pull/253 seems to be a fix for this

**Comment by jsamr** [comment]
This issue will be fixed in version 4.2.2.
A pre-release is already available if you want this fix now, see #385.
